load/UpdateDBPLPTSTable.py

```diff
@@ -339,27 +358,38 @@ def updateBibleFilesetCopyrightOrganizations(self, filesetList):
 		#unknownCopyrights = orgs.validateCopyrights()
 		orgs.update_licensors(filesetList)
 
-	def upsertBibleFileset(self, dbConn, setTypeCode, setSizeCode, filesetId, isContentLoaded=0):
+	def upsertBibleFileset(self, dbConn, setTypeCode, setSizeCode, filesetId, isContentLoaded=0, isArchived=0):
+		# Avoid creating or updating the fileset if both values are true
+		if isContentLoaded == True and isArchived == True:
+			return None
+
 		tableName = "bible_filesets"
 		pkeyNames = ("hash_id",)
-		attrNames = ("id", "asset_id", "set_type_code", "set_size_code", "content_loaded")
+		attrNames = ("id", "asset_id", "set_type_code", "set_size_code", "content_loaded", "archived")
 		updateRows = []
 		bucket = self.config.s3_vid_bucket if setTypeCode == "video_stream" else self.config.s3_bucket	
 		hashId = self.getHashId(bucket, filesetId, setTypeCode)
-		row = dbConn.selectRow("SELECT id, asset_id, set_type_code, set_size_code, content_loaded FROM bible_filesets WHERE hash_id=%s", (hashId,))
+		row = dbConn.selectRow("SELECT id, asset_id, set_type_code, set_size_code, content_loaded, archived FROM bible_filesets WHERE hash_id=%s", (hashId,))
 		if row == None:
-			updateRows.append((filesetId, bucket, setTypeCode, setSizeCode, isContentLoaded, hashId))
+			updateRows.append((filesetId, bucket, setTypeCode, setSizeCode, isContentLoaded, isArchived, hashId))
 			self.dbOut.insert(tableName, pkeyNames, attrNames, updateRows)
+			return hashId
 		elif isContentLoaded == 1:
```

Comment:
take off the elif - it is unneeded. should just be a series of if statements

Reply:
@bradflood , ok, you are right. I have updated this function and you can see it in the following line: https://github.com/faithcomesbyhearing/dbp-etl/pull/140/files#diff-a8b84dcb2a008d41509d439d0dad3855517bc429e17563a012d78d868ced3024R361